about.html

```diff
@@ -49,6 +71,7 @@
             src="./assets/images/hamburg.png"
             alt="nav-section"
             class="hamburg-icon"
+            id="menu-close-btn"
           />
         </nav>
       </section>
```

Comment:
- [ ] Kindly make sure to add some padding for the top section so it takes more space vertically as per the provided design, this will make your website much cleaner and make use of white space
- [ ] Also kindy consider removing the background color from the top section contact info part so it looks identical to the provided design

|About page|Provide design|
|-|-|
|![Screenshot_20230322_092046](https://user-images.githubusercontent.com/25227022/226857686-12226d19-bab8-4b7e-8007-49e0ba9ab0ca.png)|![Screenshot_20230322_092124](https://user-images.githubusercontent.com/25227022/226857687-85ef80e4-0529-4090-9ecd-b90812e111e9.png)|